drive/client.py

```diff
@@ -202,11 +202,11 @@ def files_shared_with_me(self) -> List[File]:
         """
         Return a list of files (and 'directories') 'shared with me'.
         """
-        return self._execute_file_request(self._files.list(q="sharedWithMe=true"))
+        return cast(List[File], self._execute_file_request(self._files.list(q="sharedWithMe=true")))
 
     def get_shared_file(self, name: str,
                         is_directory: Optional[bool] = None,
-                        raise_if_not_found=True) -> Optional[File]:
+                        raise_if_not_found: bool = True) -> Optional[File]:
```

Comment:
This `: bool` is not needed because `True` is already a `bool`.

Reply:
See above: https://github.com/bfontaine/drive/pull/13#discussion_r2015341970